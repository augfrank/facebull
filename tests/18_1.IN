M0   C0   C4  2093
M1   C0   C8  2509
M2   C0   C10  2027
M3   C0   C15  2594
M4   C0   C16  2559
M5   C1   C7  2992
M6   C1   C10  2180
M7   C1   C17  2377
M8   C2   C8  2912
M9   C2   C12  2706
M10   C3   C5  2171
M11   C3   C14  2356
M12   C4   C7  2105
M13   C4   C11  2239
M14   C5   C10  2341
M15   C5   C11  2671
M16   C6   C1  2299
M17   C6   C4  2027
M18   C6   C7  2674
M19   C6   C15  2618
M20   C7   C1  2085
M21   C7   C6  2791
M22   C7   C14  2170
M23   C7   C16  2437
M24   C8   C0  2938
M25   C8   C14  2169
M26   C9   C8  2281
M27   C9   C10  2553
M28   C9   C13  2409
M29   C9   C15  2531
M30   C10   C9  2855
M31   C10   C12  2392
M32   C10   C13  2742
M33   C11   C2  2671
M34   C12   C1  2513
M35   C13   C2  2855
M36   C13   C6  2116
M37   C14   C3  2846
M38   C14   C13  2457
M39   C15   C7  2079
M40   C15   C8  2327
M41   C15   C14  2321
M42   C16   C3  2507
M43   C16   C8  2629
M44   C16   C15  2035
M45   C17   C0  2304
M46   C17   C2  2201
M47   C17   C5  2325
M48   C17   C7  2578
M49   C17   C9  2923
