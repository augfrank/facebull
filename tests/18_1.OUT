42766
2 7 9 10 12 15 17 23 25 28 30 33 34 36 37 40 44 45
